src/zmalloc.h

```diff
@@ -55,6 +55,12 @@
 #error "Newer version of jemalloc required"
 #endif
 
+#elif defined(USE_MEMKIND)
+#define ZMALLOC_LIB "memkind"
+#include <memkind.h>
+#define HAVE_MALLOC_SIZE 1
+#define zmalloc_size(p) memkind_malloc_usable_size(NULL, p)
```

Comment:
use the APIs with the NULL kind will have the performance impact. need the performance data for the DRAM only (with kind and w NULL kind)

Reply:
@guoanwu 
Regarding memkind_malloc_usable_size and NULL kind. For default allocator used in memkind (jemalloc) there will be no performance impact for memkind_malloc_usable_size function - I will update this in memkind manual.